codegen/templates/endpoint.tmpl

```diff
@@ -146,11 +181,19 @@ func (w {{$workflow}}) Handle(
 		return {{if eq .ResponseType ""}}nil, err{{else}}nil, nil, err{{end}}
```

Comment:
Want to return partial headers here in error case if we have headers. 

Add a TODO for now ?